src/Player/PlayerDriver.cpp

```diff
@@ -62,11 +62,13 @@ void testPlayers()
     t.at(i) = NULL;
   }
   // Delete orderlist pointer
-  for (int i = 0; i < o.size(); i++)
+  for (int i = 0; i < o->getOL()->size(); i++)
   {
-    delete o.at(i);
-    o.at(i) = NULL;
+    delete o->getOL()->at(i);
+    o->getOL()->at(i) = NULL;
   }
+  o->deleteOrdersList();
```

Comment:
i removed it, and added a comment in player's MR ([here](https://github.com/LaraMo/Risk-Swift-Sloths/pull/29/files#diff-dda7ec69e3b4d201041d4261f4b19d25aa4f70ea750eb5862bc80504dfc04c09R64))
player should have its own distractor that takes care of destroying objects its controlling such as hand and orderlist. 

I have left lines 65-68 because you created orders that are on the playerDriver and they need to be destroyed by the driver.

Reply:
Ok, so maybe we should all make distructors? and Edit our own testDrivers()?